egui/src/context.rs

```diff
@@ -97,6 +100,14 @@ impl FrameState {
         self.unused_rect = Rect::nothing(); // Nothing left unused after this
         self.used_by_panels = self.used_by_panels.union(panel_rect);
     }
+
+    pub(crate) fn scroll_delta(&self) -> Vec2 {
+        self.scroll_delta
+    }
+
+    pub(crate) fn clear_scroll_delta(&mut self) {
+        self.scroll_delta = Vec2::zero();
+    }
```

Comment:
I'd prefer if `scroll_delta` was just made `pub(crate)` to make the use of it more transparent (and less code!).